[2024-04-10T09:43:28.279+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-10T09:43:28.324+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: trigger_airbyte_job_example.airbyte_check_sync manual__2024-04-10T09:43:10.396564+00:00 [queued]>
[2024-04-10T09:43:28.352+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: trigger_airbyte_job_example.airbyte_check_sync manual__2024-04-10T09:43:10.396564+00:00 [queued]>
[2024-04-10T09:43:28.353+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-10T09:43:28.415+0000] {taskinstance.py:2327} INFO - Executing <Task(AirbyteJobSensor): airbyte_check_sync> on 2024-04-10 09:43:10.396564+00:00
[2024-04-10T09:43:28.440+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=847) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-10T09:43:28.446+0000] {standard_task_runner.py:63} INFO - Started process 849 to run task
[2024-04-10T09:43:28.437+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'trigger_airbyte_job_example', 'airbyte_check_sync', 'manual__2024-04-10T09:43:10.396564+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/dag_aribyte_ex.py', '--cfg-path', '/tmp/tmpbj_79gje']
[2024-04-10T09:43:28.450+0000] {standard_task_runner.py:91} INFO - Job 20: Subtask airbyte_check_sync
[2024-04-10T09:43:28.670+0000] {task_command.py:426} INFO - Running <TaskInstance: trigger_airbyte_job_example.airbyte_check_sync manual__2024-04-10T09:43:10.396564+00:00 [running]> on host ff62e46d2b31
[2024-04-10T09:43:29.028+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='trigger_airbyte_job_example' AIRFLOW_CTX_TASK_ID='airbyte_check_sync' AIRFLOW_CTX_EXECUTION_DATE='2024-04-10T09:43:10.396564+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-10T09:43:10.396564+00:00'
[2024-04-10T09:43:29.042+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-10T09:43:29.044+0000] {baseoperator.py:399} WARNING - AirbyteJobSensor.execute cannot be called outside TaskInstance!
[2024-04-10T09:43:29.095+0000] {base.py:84} INFO - Using connection ID '***_call_airbyte' for task execution.
[2024-04-10T09:43:29.173+0000] {airbyte.py:104} INFO - Waiting for job 139 to complete.
[2024-04-10T09:44:29.183+0000] {base.py:84} INFO - Using connection ID '***_call_airbyte' for task execution.
[2024-04-10T09:44:29.225+0000] {airbyte.py:99} INFO - Job 139 completed successfully.
[2024-04-10T09:44:29.227+0000] {base.py:294} INFO - Success criteria met. Exiting.
[2024-04-10T09:44:29.227+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-10T09:44:29.245+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=trigger_airbyte_job_example, task_id=airbyte_check_sync, execution_date=20240410T094310, start_date=20240410T094328, end_date=20240410T094429
[2024-04-10T09:44:29.313+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-10T09:44:29.334+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-10T09:44:29.336+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
